salt/modules/network.py

```diff
@@ -1140,7 +1140,7 @@ def convert_cidr(cidr):
     ret = {'network': None,
            'netmask': None}
     cidr = calc_net(cidr)
-    network_info = salt.ext.ipaddress.ip_network(cidr)
```

Comment:
This is `salt.ext.ipaddress`: https://github.com/saltstack/salt/blob/develop/salt/ext/ipaddress.py - it would suggest there's a different issue if you're seeing an error when importing this module. Can you elaborate?

Reply:
Depending on the Python version we either import directly or from salt.ext. In both cases we have to use ipaddress directly, not via salt.ext.ipaddress.

Reply:
Ah alright, that makes sense. Thanks!